apps/passport-server/src/services/generic-issuance/subservices/PipelineAPISubservice.ts

```diff
@@ -520,4 +538,117 @@ export class PipelineAPISubservice {
       }
     );
   }
+
+  /**
+   * todo a comment
+   */
+  public async handleGetOfflineTickets(
+    request: PodboxGetOfflineTicketsRequest
+  ): Promise<PodboxGetOfflineTicketsResponseValue> {
+    return traced(SERVICE_NAME, "handleGetOfflineTickets", async (span) => {
+      logger(LOG_TAG, "handleGetOfflineTickets", str(request));
+
+      let emailAddress;
+      try {
+        emailAddress = (
+          await this.credentialSubservice.verifyAndExpectZupassEmail(
+            request.credential
+          )
+        ).emailClaim.emailAddress;
+      } catch (_e) {
```

Comment:
```suggestion
      } catch {
```

Reply:
Oh wow, I did not know that this is valid syntax.